lang/cs/Org.Apache.REEF.Driver/Evaluator/EvaluatorException.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I believe this style is the result of configuring ReSharper to automatically insert a license into each file.

I don't think we have a convention. Should we have one?